defaults/main.yml

```diff
@@ -34,3 +34,16 @@ couchdb_pki_cert_suffix: '.pem'
 couchdb_init: systemd
 couchdb_activate: True
 couchdb_async_io_threads: 16
+
+# couchdb_cluster_settings:
+#   q=8
```

Comment:
Sorry about that. I fixed that in my settings in my teams branch but forgot to push it here https://github.com/andrewrothstein/ansible-couchdb-cluster/pull/2/commits/6e42f4fcbf7fa96d3369db0e92255b8c67f55d14